### PR TITLE
logger

### PR DESCRIPTION
**Pull request by YaTut1901**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
-target/*
\ No newline at end of file
+target/*
+logs/*
```
Comment (andriylagun) [inline .gitignore]: empty line

File: src/main/java/main/movie/dao/impl/CinemaHallDaoImpl.java [hunk 2]
```diff
@@ -6,14 +6,19 @@
...
 import main.movie.model.CinemaHall;
 import main.movie.util.HibernateUtil;
+import org.apache.log4j.Logger;
 import org.hibernate.Session;
 import org.hibernate.Transaction;
 import org.hibernate.query.Query;
 
 @Dao
 public class CinemaHallDaoImpl implements CinemaHallDao {
+
```
Comment (dmytro-hryhoruk) [inline src/main/java/main/movie/dao/impl/CinemaHallDaoImpl.java]:
```suggestion
```

File: src/main/java/main/movie/dao/impl/MovieDaoImpl.java [hunk 3]
```diff
@@ -6,15 +6,19 @@
...
 import org.hibernate.query.Query;
 
 @Dao
 public class MovieDaoImpl implements MovieDao {
 
+    private static final Logger logger = Logger.getLogger(CinemaHallDaoImpl.class);
+
     @Override
     public Movie add(Movie movie) {
+        logger.info("Movie creating...");
```
Comment (dmytro-hryhoruk) [inline src/main/java/main/movie/dao/impl/MovieDaoImpl.java:21]: let's add loggin about successfully added object(after it was added)

**Review by dmytro-hryhoruk — COMMENTED** [review]
apply comment to all dao layers

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**